src/app/shared/utils/generate-id.ts

```diff
@@ -0,0 +1,13 @@
+/**
+ * Generate a random id with the specified length.
+ * The id contains only lowercase letters and numbers.
+ *
+ * @param length The length of the id.
+ */
+export function generateId(length: number): string {
+  let result = '';
+  do {
+    result += Math.random().toString(36).substring(2, 9);
+  } while (result.length < length);
+  return result.substring(0, length);
+}
```

Comment:
If the needed `length` is longer than the result of the random number generator, I need to generate a second or third number. That's why I did it like this. I didn't wanna use the `crypto` module or a `uuid` package, since I need a specific format for this ID (more on why exactly this format will be clear in a future feature).